locksmith/src/routes/v2/claim.ts

```diff
@@ -1,18 +1,11 @@
 import express from 'express'
 import { claim } from '../../controllers/v2/claimController'
-import { authenticatedMiddleware } from '../../utils/middlewares/auth'
 import { createGeoRestriction } from '../../utils/middlewares/geoRestriction'
 
 const router = express.Router({ mergeParams: true })
 
 // Disallow claim due to spam and bot activity
 const geoRestriction = createGeoRestriction(['RU', 'UA'])
 
-router.post(
-  '/:network/locks/:lockAddress',
-  geoRestriction,
-  authenticatedMiddleware,
-  claim
-)
-
+router.post('/:network/locks/:lockAddress', geoRestriction, claim)
```

Comment:
Removed `authenticatedMiddleware` since this API call can now support walletless airdrops.

Reply:
Use the rate limit middleware to at least rate limit. I think it would be good idea.

Reply:
Added!